miasm2/os_dep/win_api_x86_32.py

```diff
@@ -1411,6 +1456,16 @@ def kernel32_lstrcpyA(jitter):
 def kernel32_lstrcpy(jitter):
     my_strcpy(jitter, whoami(), jitter.get_str_ansi, jitter.set_str_ansi)
 
+def msvcrt__mbscpy(jitter):
+    ret_ad, args = jitter.func_args_cdecl(["ptr_str1", "ptr_str2"])
+    s2 = jitter.get_str_unic(args.ptr_str2)
+    print("mbscpy '%s'" % s2)
```

Comment:
You have made an effort to use logging facilities everywhere else, too bad to use a `print` here

Reply:
It's because I missed this one :)